src/ray/object_manager/object_store_notification_manager.cc

```diff
@@ -85,9 +96,13 @@ void ObjectStoreNotificationManager::ProcessStoreLength(
 void ObjectStoreNotificationManager::ProcessStoreNotification(
     const boost::system::error_code &error) {
   if (error) {
-    RAY_LOG(FATAL)
-        << "Problem communicating with the object store from raylet, check logs or "
-        << "dmesg for previous errors: " << boost_to_ray_status(error).ToString();
+    if (exit_on_error_) {
+      RAY_LOG(FATAL)
```

Comment:
Is there a reason we `RAY_LOG(FATAL)` here, but `_exit` in the other code block?

Reply:
i don't know the reason